installer/options.go

```diff
@@ -79,12 +85,17 @@ func NewAuthzOptions() AuthzOptions {
 }
 
 func (o *AuthOptions) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&o.VerbosityLevel, "v", o.VerbosityLevel, "Log level for V logs")
 	fs.StringVar(&o.PkiDir, "pki-dir", o.PkiDir, "Path to directory where pki files are stored.")
 	fs.StringVarP(&o.Namespace, "namespace", "n", o.Namespace, "Name of Kubernetes namespace used to run guard server.")
 	fs.StringVar(&o.Addr, "addr", o.Addr, "Address (host:port) of guard server.")
 	fs.BoolVar(&o.RunOnMaster, "run-on-master", o.RunOnMaster, "If true, runs Guard server on master instances")
 	fs.StringVar(&o.PrivateRegistry, "private-registry", o.PrivateRegistry, "Private Docker registry")
 	fs.StringVar(&o.imagePullSecret, "image-pull-secret", o.imagePullSecret, "Name of image pull secret")
+	fs.StringVar(&o.HttpsProxy, "proxy-https", o.HttpsProxy, "Https proxy URL to be used")
+	fs.StringVar(&o.HttpProxy, "proxy-http", o.HttpProxy, "Http proxy URL to be used")
+	fs.StringVar(&o.NoProxy, "proxy-skip-range", o.NoProxy, "List of URLs/CIDRs for which proxy should not to be used")
+	fs.StringVar(&o.ProxyCert, "proxy-cert", o.ProxyCert, "Path to the certificate file for proxy")
```

Comment:
should we add validation that proxy works only with azure and once other folks adds support they can remove themselves from validation. Once we assume that proxy will start failing for other providers if customer tries that.

Reply:
not sure whether it will fail with others though

Reply:
should we discuss this with tamal when we open it on upstream?

Reply:
it will right.. if some one passes those proxy values for github, if github uses http client and it does not have proxy implementation, it will fail, no?

Reply:
fine with asking him if we need to add validation or not?